docker-multinode/common.sh

```diff
@@ -232,12 +227,14 @@ kube::multinode::restart_docker(){
         yum -y -q install bridge-utils
       fi
 
-      kube::helpers::file_replace_line ${DOCKER_CONF} `# Replace content in this file` \
-        "--bip" `# Find a line with this content...` \
-        "OPTIONS=\"\$OPTIONS --mtu=${FLANNEL_MTU} --bip=${FLANNEL_SUBNET}\"" `# ...and replace the found line with this line`
+      # Is there an uncommented OPTIONS line at all?
+      if [[ -z $(grep "OPTIONS" $DOCKER_CONF | grep -v "#") ]]; then
+        echo "\$OPTIONS=\"--mtu=${FLANNEL_MTU} --bip=${FLANNEL_SUBNET}\"" >> $DOCKER_CONF
```

Comment:
This means that these options won't be appended to any existing option, but instead replace them. Maybe that's not a problem, but I'd suspect it might be.